utils/multer.js

```diff
@@ -0,0 +1,44 @@
+const multer = require('multer')
+const multerConstant = require('../constants/multer')
+const multerMemoryStorage = multer.memoryStorage()
+
+const MB_1 = multerConstant.FILE_SIZE_1MB
+const profileFileSize = multerConstant.PROFILE_FILE_SIZE
+
+function fileFilterImagesOnly (req, file, cb) {
+  const mimetype = file.mimetype
+  const isMimeTypePng = (mimetype === 'image/png')
+  const isMimeTypeJpeg = (mimetype === 'image/jpeg')
+  if (isMimeTypePng || isMimeTypeJpeg) {
+    return cb(null, true)
+  }
+  return cb(new multer.MulterError('TYPE_UNSUPPORTED_FILE'), false)
+};
```

Comment:
Rest of the functions in the same file are arrow functions and this one is not. By mistake?